CMakeSettings.json

```diff
@@ -22,7 +22,7 @@
       "buildCommandArgs": "",
       "ctestCommandArgs": "",
       "inheritEnvironments": [ "msvc_x64_x64" ],
-      "addressSanitizerEnabled": true
+      "addressSanitizerEnabled": false
```

Comment:
why is address sanitizer getting disabled? any reasons for this?

Reply:
Is only releated to MSVC (Visual Studio Builds). If enabled abseil does not build as it wants its own address sanitizer